app/assets/javascripts/bp_ontolobridge.js

```diff
@@ -202,8 +202,11 @@ function clearStatusMessages() {
 
 function showStatusMessages(success, error) {
   if (success.length > 0) {
-    jQuery("#ob_success_message").html(success);
-    jQuery("#ob_success_message").show();
+      let ob = jQuery("#ob_success_message")
+      console.log('show ob message')
+      console.log(ob)
+      ob.html(success);
+      ob.show();
```

Comment:
Yes  it's just a cleanup this is not related at all with the administration functionality.
I noticed that all the JS part was using the old syntaxe of JS (before 2015) with **var** (unstead of let or const ) and **object.prototype** (unstead of decalaring a class) , it will be a good thing to upgrade the frontend part (may be it's already in your roadmap) cause as of Rails 6, Webpack(er) replaces Sprockets as the new standard for writing Javascript in Rails applications, it will permit to use the last apport of es6 like declaring classes (and OOP); create modules ,export/import them wich will permit to better organize the code.